pandas/core/strings.py

```diff
@@ -2283,6 +2196,8 @@ def cat(self, others=None, sep=None, na_rep=None, join=None):
 
         if isinstance(others, compat.string_types):
             raise ValueError("Did you mean to supply a `sep` keyword?")
+        if sep is None:
```

Comment:
Not anything you need to change in this PR but if we just overwrite `None` here it would probably make more sense to change the default function signature to simply be `''`